libs/types/src/cdf/cast-vote-records/vx-schema.json

```diff
@@ -244,7 +244,7 @@
       "type": "object"
     },
     "CVR.CVRContestSelection": {
-      "required": ["@type", "SelectionPosition"],
+      "required": ["@type", "SelectionPosition", "ContestSelectionId"],
```

Comment:
Rationale: we have no use for a contest selection if we cannot tie it to an ID in the election definition.